### PR TITLE
Removed signup copy changes to step 1 a/b test code

### PR DESCRIPTION
**Pull request by fditrapani**
This PR removes the signup copy a/b test on step 1 of the Calypso Signup Flow. Our test beat out the control and as a result we'll need to get some new strings translated. This PR will not merge until the translations have been included and our circelci tests pass. 

cc @markryall @yoavf

### CODE REVIEW

File: client/signup/steps/design-type-with-store/index.jsx [hunk 1]
```diff
@@ -34,34 +33,28 @@ class DesignTypeWithStoreStep extends Component {
...
+			gridText = translate( 'To present your creative projects in a visual showcase.' ),
+			storeText = translate( 'To sell your products or services and accept payments.' );
 
 		return [
-			{ type: 'blog', label: translate( 'A list of my latest posts' ), image: <BlogImage /> },
-			{ type: 'page', label: translate( 'A welcome page for my site' ), image: <PageImage /> },
-			{ type: 'grid', label: translate( 'A grid of my latest posts' ), image: <GridImage /> },
-			{ type: 'store', label: translate( 'An online store' ), image: <StoreImage /> },
+			{ type: 'blog',
+				label: translate( 'Start with a blog' ),
```
Comment (a8ci18n) [inline client/signup/steps/design-type-with-store/index.jsx]: Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Start with Bluehost' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/remove-signup-test)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Comment (fditrapani) [inline client/signup/steps/design-type-with-store/index.jsx]: 👎

**Comment by fditrapani** [comment]
@yoavf I've had this PR up for a week. When I last checked, it was out of sync with master and at 77%. After rebasing, it looks like we're back down to 0%. Any idea of how long this might take?

**Comment by yoavf** [comment]
@fditrapani thanks for the ping - the 0% was a bug introduced yesterday. The numbers will update in a couple of minutes. I'll check what's holding up the translations.

**Comment by fditrapani** [comment]
Thanks @yoavf!

**Comment by yoavf** [comment]
@fditrapani we're just missing one locale (Arabic) - Hopefully we'll get it by tomorrow. We've had some deployment issues on some of the locales, so I'll double check that we're actually ready tomorrow.

**Comment by fditrapani** [comment]
Cool, thanks for looking into that for me @yoavf 👊

**Comment by yoavf** [comment]
@fditrapani translation is done, you're good to go :)

**Comment by fditrapani** [comment]
Thanks @yoavf!

**Comment by fditrapani** [comment]
@markryall can you review this PR from a code perspective?

**Comment by markryall** [comment]
this looks good to me - i'm not sure i'd add the eslint exceptions but it isn't a show stopper.

File: client/signup/steps/design-type-with-store/index.jsx [hunk 2]
```diff
@@ -34,34 +34,28 @@ class DesignTypeWithStoreStep extends Component {
...
-					image: <GridImage /> },
-				{ type: 'store',
-					label: 'an online store',
-					description: 'To sell your products or services and accept payments.',
-					image: <StoreImage /> },
-			];
-		}
+		const blogText = translate( 'To share your ideas, stories, and photographs with your followers.' ), // eslint-disable-line max-len
+			siteText = translate( 'To promote your business, organization, or brand and connect with your audience.' ), // eslint-disable-line max-len
+			gridText = translate( 'To present your creative projects in a visual showcase.' ),
```
Comment (markryall) [inline client/signup/steps/design-type-with-store/index.jsx]: it is more conventional to have separate declarations per line.

maybe this kind of format would sort out the eslint issue:

```
const varName = translate(
  '...'
);
```

**Pull request merged**